HousingRepairsOnlineApi.Tests/HelpersTests/RepairDayWindowHelperTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.Helpers;
+using Xunit;
+
+namespace HousingRepairsOnlineApi.Tests.HelpersTests
+{
+    public class RepairDayWindowHelperTests
+    {
+        private RepairDayWindowHelper systemUnderTest;
+
+        private readonly Repair _repair = new();
+
+        public RepairDayWindowHelperTests()
+        {
+            IEnumerable<RepairDayWindow> repairDayWindows = new List<RepairDayWindow>()
+            {
+                new() { NumberOfDays = 3, Priority = "2" },
+                new() { NumberOfDays = 30, Priority = "3" }
+            };
+            systemUnderTest = new RepairDayWindowHelper(repairDayWindows);
+        }
+
+        [Fact]
+#pragma warning disable CA1707
+        public void GivenRepairDayWindows_WhenCallingGetDaysForRepair_ReturnsCorrectDays()
+#pragma warning restore CA1707
+        {
+            // Arrange
+            _repair.Priority = "2";
+
+            // Act
+            var result = systemUnderTest.GetDaysForRepair(_repair);
+
+            // Assert
+            Assert.Equal(3, result);
+        }
+
+        [Fact]
+#pragma warning disable CA1707
+        public void GivenRepairDayWindows_WhenCallingGetDaysForRepairForSecondWindow_ReturnsCorrectDays()
+#pragma warning restore CA1707
+        {
+            // Arrange
+            _repair.Priority = "3";
+
+            // Act
+            var result = systemUnderTest.GetDaysForRepair(_repair);
+
+            // Assert
+            Assert.Equal(30, result);
+        }
+
+        [Fact]
+#pragma warning disable CA1707
+        public void GivenRepairDayWindows_WhenCallingGetDaysForRepairForSecondWindowWithManyResults_ReturnsFirstCorrectDays()
+#pragma warning restore CA1707
+        {
+            // Arrange
+            RepairDayWindowHelper repairDayWindowHelper = new RepairDayWindowHelper(new List<RepairDayWindow>()
```

Comment:
Nitpick: an optimisation would be to use an array as it will only allocate the memory required and doesn't allow additional values to be added, which is fine in this case.
```suggestion
            RepairDayWindowHelper repairDayWindowHelper = new RepairDayWindowHelper(new RepairDayWindow[]
```